store.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>Karma store</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="./css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+
+  <body>
+
+    <header class="store-header">
+
+        <img class="svg-logo"
+            src="./img/karma-logo.svg" 
+            alt="Karma company logo">
+        </img>
+        <nav class="nav-bar">
+            <ul class="navigation__list">
+                <li class="active">Meet Karma</li> 
```

Comment:
Is this active class still on the correct element?

Reply:
I used this class to make one element("Meet Karma") bold as shown on the sample picture.

Reply:
ah, to me this classname suggested that this page is the current (active) page. So when you are on the `Store` page the store link might be active.